src/metrics.rs

```diff
@@ -34,6 +34,13 @@ pub static CMD_TOP_COUNTER: Lazy<BothModesCounters> = Lazy::new(|| {
         inline: Counter::new("command_top (inline)", opts.const_label("mode", "inline")),
     }
 });
+pub static CMD_LOAN_COUNTER: Lazy<BothModesCounters> = Lazy::new(|| {
```

Comment:
Actually, this is a complex both modes counter. We should count an attempt to get a loan and the consent separately.